doc/ws.md

```diff
@@ -456,7 +456,7 @@ Removes an event listener emulating the `EventTarget` interface.
 
 ### websocket.send(data[, options][, callback])
 
-- `data` {Any} The data to send.
+- `data` {string | Buffer} The data to send.
```

Comment:
This is not correct, you can also pass a number, and anything accepted by `Buffer.from()`

Reply:
Apologies, you're totally right. I'll get that updated.

Reply:
Though objects are accepted by `Buffer.from()`, but not `socket.send()`.

Reply:
It does not matter. If possible, the `data` argument is converted to `Buffer` before being sent.

Reply:
Ah, yeah, I had something odd I was trying to use. I didn't look at the specifics for the implementations.

Reply:
Hopefully that update is (more) correct. It's `number` + all the `Buffer.from()` args. Apologies if it's not, still super new to the lib, but the `Any` threw me for a loop.